site/src/component/ReviewItemGrid/ReviewItemGrid.scss

```diff
@@ -0,0 +1,14 @@
+.reviews-grid {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(500px, 1fr));
+  gap: 1rem;
+}
+
+.reviews-grid > div {
+  width: 100%;
+  height: 100%;
+}
+
+.reviews-grid > div > * {
+  height: 100%;
+}
```

Comment:
I believe you could instead just use the `justify-items` and `align-items` properties instead of setting them on the child. Also, what is the direct child `div` for?